### PR TITLE
Create EnergyX 0xbbe899c61198d1826a43e20ea19efc46e50c2b00 Logo

### PR DESCRIPTION
**Pull request by oceyx**
logo

#8021

### CODE REVIEW

**Comment by Cryptocool1** [comment]
nfo file should be named info.json
Logo file should be named logo.png
Please rename

File: blockchains/smartchain/assets/0xbbe899c61198d1826a43e20ea19efc46e50c2b00 [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "name": "Safe Energy",
+    "type": "BEP20",
+    "symbol": "EnergyX",
+    "decimals": 8,
+    "website": "https://safeenergy.club/",
+    "description": "SAFEENERGY,  $ENERGYX is a community driven deflationary charity token with future projects to reduce our carbon footprint by making renewable energy sources more accesible. $ENERGYX is on Binance Smart Chain (BSC) with a max supply of 1 Quadrillion.",
+    "explorer": "https://bscscan.com/token/0xbbe899c61198d1826a43e20ea19efc46e50c2b00",
+    "status": "active",
+    "id": "0xbbe899c61198d1826a43e20ea19efc46e50c2b00"
+
```
Comment (oceyx) [inline blockchains/smartchain/assets/0xbbe899c61198d1826a43e20ea19efc46e50c2b00:11]: "logo": "https://raw.githubusercontent.com/Safe-Energy/BEP20Token.sol/main/logo.png"

**Review by oceyx — COMMENTED** [review]
Logo.png added

**Comment by oceyx** [comment]
Help me? tg: @oceyx

**Comment by oceyx** [comment]
> nfo file should be named info.json
> Logo file should be named logo.png
> Please rename

Please help me. ceylan.oguzhan@yandex.com

**Comment by Cryptocool1** [comment]
hello,
you added only file with info

**Comment by Cryptocool1** [comment]
Please come back to us when all criteria are met.
When creating a new PR make sure to mention this PR, so that we can see that fee has been paid already here.
You may use our Assets app (in beta); close this PR; go to https://assets.trustwallet.com, log on with Github, enter contract ID, logo and some additional info, and it will create a PR for you. See https://community.trustwallet.com/t/how-to-submit-a-token-logo-using-assets-app/82957\

**Pull request closed without merging**